packaging/helm/README.md

```diff
@@ -0,0 +1,11 @@
+[Helm](helm.sh) is a package management system for kubernetes applications.
+
+Once you have Helm installed, you can install the arlas-server chart by performing the following:
+
+```bash
+helm install -f my_values.yaml arlas-server
+```
+
+, where `my_values.yaml` contains custom configuration settings for the chart.
```

Comment:
Perhaps you should also add a new section in docs/arlas-server-configuration.md about this conf file?

Reply:
Done in 1af5c55bf439f7e44c5cbf095a20485c03a74bc5, see [here](https://github.com/gisaia/ARLAS-server/blob/1af5c55bf439f7e44c5cbf095a20485c03a74bc5/docs/arlas-server-configuration.md#helm-configuration).